packages/clerk-auth/package.json

```diff
@@ -38,16 +38,22 @@
   },
   "homepage": "https://github.com/honojs/middleware",
   "peerDependencies": {
-    "@clerk/backend": ">=0.30.0 <1",
+    "@clerk/backend": "1",
```

Comment:
Maybe I'm missing something, but I thought `1` was supposed to mean any version with a restriction to a major `1` (i.e. `1.0.0` and `1.2.3`, but not `2.0.0` or `0.5.0`), which makes sense to me since we want to pin it to a current major version.

Your suggested semver `>=1.0.*` actually matches any version (including major ones like `2` and so on), starting from `1.0.0`. I suppose we can't be sure next major versions of Clerk SDK will be supported, right? Please correct me if I'm wrong.

Reply:
Yeah you are right, sorry this should be `^1.0.0`not `>=1.0.*`